MatrixSDK/Crypto/MXCrypto.m

```diff
@@ -1051,16 +1053,11 @@ - (void)flushNewDeviceRequests
         // tight-loop.
         for (NSString *userId in usersDevicesInfoMap.userIds)
         {
-            [pendingNewDevices removeObjectsForUser:userId];
+            [pendingUsersWithNewDevices removeObject:userId];
```

Comment:
Agreed :/
The code changed to this in order to fix another race where the user sends a message whereas the code is downloading a new device key. (https://github.com/matrix-org/matrix-ios-sdk/pull/196)